src/BoxAnnotations.ts

```diff
@@ -3,6 +3,7 @@ import BaseAnnotator from './common/BaseAnnotator';
 import ImageAnnotator from './image/ImageAnnotator';
 import DocumentAnnotator from './document/DocumentAnnotator';
 import { IntlOptions, Permissions, PERMISSIONS, Type } from './@types';
+import './utils/polyfill';
```

Comment:
Should the polyfill be the first import?

Reply:
Yeah it makes sense.